cluster/terraform_kubernetes/alertmanager.tf

```diff
@@ -0,0 +1,165 @@
+data "azurerm_key_vault" "key_vault_slack" {
```

Comment:
data for kv is already called in statuscake.tf so this can be removed

Reply:
fixed

Reply:
removed this